runtime/jailer.go

```diff
@@ -81,6 +83,10 @@ func newJailer(
 		return newNoopJailer(ctx, l, service.shimDir), nil
 	}
 
+	if err := os.MkdirAll(ociBundlePath, 0644); err != nil {
```

Comment:
I also think this should be 0700 for same reasons as the other comment

Reply:
Fixed!